locator/snitch_base.hh

```diff
@@ -48,6 +48,7 @@ struct snitch_config {
 
     // GCE-specific
     sstring gce_meta_server_url = "";
+    bool broadcast_rpc_address_specified_by_user = false;
```

Comment:
This is not GCE specific as the comment above suggests, right?
Maybe add a comment above: `// AWS specific`.

Reply:
This comment indeed looks confusing. But the field is not AWS specific, it is now only used by AWS snitch though. I moved the field around. Thanks.